source/enterprise/tech-guide/installation-guide.md

```diff
@@ -9,48 +9,68 @@ This guide will cover the Shopware Enterprise Dashboard installation process
 
 <div class="toc-list"></div>
 
+## System Requirements
 
-## Dependencies - Enterprise Dashboard host
+The following system libraries/applications are required to install and run the Shopware Enterprise Dashboard:
 
-The following system libraries/application are required to install and run the Shopware Enterprise Dashboard from a release package:
+##### Requirements
 
 - Linux operating system
-- PHP 5.6 or later (PHP 7 recommended) including CLI support
+- PHP 5.6 or later (PHP 7 recommended) including CLI support and mod_rewrite
 - Apache2 web server
 - MySQL
-- Ansible Version 1.9+
-- Beanstalk Version 1.9+
+- Ansible Version 2.0+
+- Beanstalk Version 1.4+, with a max job size of at least `65533 Byte`
 - Supervisor 3.0+ 
-
-If you are setting up a development version of the EDB, you might need the following additional tools:
-- Ant
-- NodeJS + NPM
-  - Grunt
-  - Bower
-  - Jasmine
-- Vagrant + Virtualbox
-
-Similar architectures might also work, but are not officially supported.
+- A distinct shell user for SSH access to the connected shop servers 
+- The EDB requires to be run from the root of an apache host, prepare a domain
+- Set PHP *(cli & apache2)* and MySQL timezone to UTC
 
 ## Installing the Shopware Enterprise Dashboard
 
-### Using the installation package
-
-For evaluation, testing or production purposes, it's recommended using one of the available installation packages:
 - Download the installation package
 - Extract the archive to your web server's content folder
-- Enable web server access to _APPLICATION_/web
-- Create a user account to execute your `supervisor` tasks (by default is set to `edb-supervisor`). It should be have permissions to connect to your Shopware servers using SSH. 
+- Enable web server access to *_APPLICATION_/web* 
 - Execute `php setup.phar` through the command line
 - Follow the instructions
-- Symlink _APPLICATION_/supervisord/edb.conf.dist into your `supervisor` to configuration folder. Keep in mind that supervisor might be configured to only include file that end in `.conf`.
+- Remember to symlink the Supervisor configuration in *_APPLICATION_/supervisord/edb.conf.dist* to enable the background processes
+
+## HowTo: Setup the background processes on Ubuntu 14.04
```

Comment:
Kannst du da vll. eben in ein, zwei Sätzen ausführen, WARUM man das braucht? Also Supervisor / Worker, asynchrones Processing - das macht es doch leichter zu verstehen, was man da gerade einrichtet :)